modules/nw-ingress-setting-a-custom-default-certificate.adoc

```diff
@@ -71,5 +71,24 @@ map[name:custom-certs-default]
 +
 The certificate secret name should match the value used to update the CR.
 
+Your Ingress Controller now uses the custom certificate, but it is not trusted
+by the web console. You must expose your CA certificate so the Ingress
+Controller can communicate with all routes.
```

Comment:
Your Ingress Controller now uses the custom certificate to secure client-side connections using TLS. Some platform components (i.e. web console) use routes for communication and therefore must trust the Ingress Controller's  custom certificate. This is accomplished through the `trustedCA` field of the `Proxy` resource. The `trustedCA` field is a reference to a `ConfigMap` containing the custom certificate and key pair used by the Ingress Controller.

@Miciah @bparees a `Secret` is used by the router and a `ConfigMap` is used for cluster-wide proxy. I think it would provide a better user experience if we settled on one resource type.

Reply:
we chose a configmap for trusted CA because CAs are public information that does not need to be protected.